delatore/configuration/resources/sources.yaml

```diff
@@ -29,22 +29,28 @@
         metric_id: diskio
         query: "SELECT LAST(writes) FROM {entity} WHERE host=~/scn3/ LIMIT 1;"
         timeout: 300
-  publishes: 'INFLUX'
+  publishes:
+    - 'INFLUX_tg'
```

Comment:
Topic should be less specific about it future subscribers, e.g. `INFLUX_INFO`, `INFLUX_DEBUG` or `INFLUX_ALL`, `INFLUX_CHANGES`